dos-report.sh

```diff
@@ -23,11 +23,14 @@ fi
 ## setup window interval for query
 window_interval="10s" 
 window_interval_long="10s"
+oversize_window=$(echo "${DURATION}+300" | bc)
```

Comment:
I like bc better because it is more general for calculation. 
For example, if I give DURATION=10.88, the $((${DURATION}+300)) does not work.
Sometimes, I use expression,  when something simple as +1

Reply:
about `bc`, got you